packages/components/src/Text/Text.tsx

```diff
@@ -2,6 +2,7 @@ import React, { ReactNode } from "react";
 import { Typography, TypographyOptions } from "../Typography";
 
 interface TextProps {
+  numberOfLines?: number;
```

Comment:
@eddysims / @chris-at-jobber What do you think of for a name for this prop. I'm sort of leaning towards `maxLines`. As it reflects that this will be the maximum number of lines to be shown before it truncates?

Reply:
Oh, I thought we wanted to make the api close to what React native has for Text component. https://reactnative.dev/docs/0.9/text#numberoflines

Reply:
I think if it was specifying the number of lines regardless of content I'd agree. @hosseintalebi it's my understanding that we'll be wrapping a lot of the react native components as well to provide more opinionated / sensible defaults on them?

If that's the case I think I'd still prefer `maxLines` or something more indicative of the behaviour here. 😄

Reply:
So something like `maxLines = single | small | large ` same as [here](https://github.com/GetJobber/jobber-mobile/blob/192507714699035d7a6ff83d1e10f5524268e1e5/src/components/JobberText/JobberText.tsx#L34) ?

Reply:
@seanhealy

Reply:
+1 for some presets for the lengths rather than an open ended `number` prop

Reply:
i think something like

```
maxLines: 'single' | 'small' | 'base' | 'large' | 'larger'
```

Or a subset of those would make sense.